sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/base_client_async.py

```diff
@@ -70,13 +69,13 @@ def _create_pipeline(self, credential, **kwargs):
         # type: (Any, **Any) -> Tuple[Configuration, Pipeline]
         self._credential_policy = None
         if hasattr(credential, 'get_token'):
-            self._credential_policy = AsyncBearerTokenCredentialPolicy(credential, STORAGE_OAUTH_SCOPE)
+            self._credential_policy = AsyncStorageBearerTokenCredentialPolicy(credential)
```

Comment:
As discussed, let's not take this specific change to avoid introducing Bearer challenges to other packages. Looks like you may have just missed reverting this for async?

Applies to file-share and queue as well.